worker/worker.go

```diff
@@ -78,7 +78,7 @@ func processCampaign(c *models.Campaign) {
 			t,
 			c.URL + "?rid=" + t.RId,
 			c.URL + "/track?rid=" + t.RId,
-			"<img src='" + c.URL + "/track?rid=" + t.RId + "'/>",
+			"<img style="display: none" src='" + c.URL + "/track?rid=" + t.RId + "'/>",
```

Comment:
Definitely. You'll see the travis ci build fails with a syntax error when using double quotes. You'll need to use single quotes or escape them with a backslash "\".